e2e-tests/functions

```diff
@@ -92,14 +104,14 @@ delete_pmm_api_key() {
 deploy_minio() {
 	local access_key
 	local secret_key
-	access_key="$(kubectl -n "${NAMESPACE}" get secret minio-secret -o jsonpath='{.data.AWS_ACCESS_KEY_ID}' | base64 -d)"
-	secret_key="$(kubectl -n "${NAMESPACE}" get secret minio-secret -o jsonpath='{.data.AWS_SECRET_ACCESS_KEY}' | base64 -d)"
+	access_key="$(kubectl -n "${OPERATOR_NS:-$NAMESPACE}" get secret minio-secret -o jsonpath='{.data.AWS_ACCESS_KEY_ID}' | base64 -d)"
```

Comment:
I don't think we deploy Minio in operator namespace so this is probably wrong.

Reply:
[1eb58d9](https://github.com/percona/percona-server-mysql-operator/pull/673/commits/1eb58d9bed545ed7f16146e9f74e7f038c08100e)